README.md

```diff
@@ -26,6 +28,21 @@ Many features normally considered fundamental by deck.gl applications are not ev
 - No Performance Optimizations
 - etc
 
+## Supported Platforms
+
+### iOS
+
+- iOS12+ is required when building for a device
+- iOS13+ is required when building for a simulator, as `Metal` is available on the simulator starting with iOS13
+- Currently has to be built with bitcode disabled (under investigation)
+
+### macOS
+
+- macOS10.14+
+
+#### Linux
+
+- Tested and runs on Ubuntu Bionic
```

Comment:
Well, not sure we have graphics working on Ubuntu. I think we need to do something to switch Ubuntu from Vulkan to OpenGL to get it to render.

Reply:
It shouldn't be using Vulkan to begin with according to build settings, so I'm kind of confused about that one.

We could likely figure this one out pretty quickly, either if you have some spare time to take a look, or I could set up a Linux box once we start wrapping things up